doc/en/user/source/data/cascaded/wms.rst

```diff
@@ -62,6 +62,18 @@ Connecting a remote WMS allows for the following features:
 
 * Full **REST Configuration**. See the :ref:`rest` section for more information about the GeoServer REST interface.
 
+Cascaded WMS Settings
+---------------------
+
+Making use of Remotely advertised styles and supported image formats.
+
+* **Remote Styles Configuration**. Remote Styles advertised in WMS capability document under <Layer> tag, can also be used. A default style and additionally supported styles can be selected. By default no remote style is selected which indicates Geoserver to use whatever style is configured remotely and all available styles are selected. This means that remote styles can be passed in a GetMap request just like local styles. If the styles on remote WMS server have changed, please re-save the layer from UI.
```

Comment:
* additionally -> additional
* no remote style is selected, which will make GeoServer use the default remote style, and makes all remote styles available for GetMap usage.. This means...
* If the styles on the remote WMS server have changed --> wait… if nothing was selected locally I hope that's not necessary though?

Reply:
yes for legacy behavior nothing can be set as forced remote style.

Reply:
if nothing was selected then its not necessary